AustinKellar/PowerOn-Prototype
# Refactor

Goal: Clean up the code while keeping the game's behavior the same

Files changed:
- BranchSpawner.cs
- BranchScript.cs

Code improvements:
- Instead of using leftBranch as a boolean variable signifying whether the branch growing is on the left side, use an enum to keep track of which tree (Left or Right) was clicked on
- Instead of using isClicked to keep track of whether or not branches should keep growing, destroy the script when the user stops holding the mouse button
- Removed commented out code
- Removed Debug.Log statements
- Moved all Scripts into the Scripts folder
- Fixed a spelling mistake: BranchSpawnner --> BranchSpawner
- Renamed a few variables for clarity

How to test: 
- Pull the refactor branch (git fetch && git checkout refactor)
- Make sure that the branches behave the same way they do on the develop branch
- Look through the 2 .cs files I changed and look for any further improvements that can be made
- If you think this PR should be merged into develop, click Merge, otherwise leave comments and ping me on Slack, then I will address the comments.
